packages/parse5/package.json

```diff
@@ -27,7 +27,15 @@
         "serialize"
     ],
     "license": "MIT",
-    "main": "dist/index.js",
+    "main": "dist/cjs/index.js",
+    "module": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "exports": {
+        ".": {
+            "import": "dist/index.js",
+            "require": "dist/cjs/index.js"
+        }
+    },
```

Comment:
this breaks [rehype/test/parse-error.js](https://github.com/rehypejs/rehype/blob/620bb9cdf53a4dbe70adfca8d66c8824eb3f513f/test/parse-error.js#L7)

```js
import p5errors from 'parse5/lib/common/error-codes.js'
```

wontfix in rehype, as [ERR](https://github.com/inikulin/parse5/blob/075be2e712644410e1ceae67ebabad958bc0b66e/packages/parse5/lib/common/error-codes.ts#L9) is not exported by parse5

&rarr; export as ErrorCodes?

https://nodejs.org/api/esm.html

> module files within packages can be accessed by appending a path to the package name **unless the package's [package.json](https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions) contains an ["exports"](https://nodejs.org/api/packages.html#exports) field, in which case files within packages can only be accessed via the paths defined in ["exports"](https://nodejs.org/api/packages.html#exports).**

Reply:
the package exports are right IMO. if we have consumers of the error codes enum, we should probably just export it top-level along with `ParserError`, possibly as a useful name like you said (`ErrorCodes`).

i.e. in `index.ts`, `export {ERR as ErrorCodes}` or something